src/Chapter3.hs

```diff
@@ -903,13 +986,38 @@ We will call such a typeclass "Append". You can find its definition below.
 Implement instances of "Append" for the following types:
 
   ✧ The "Gold" newtype where append is the addition
-  ✧ "List" where append is list concatenation
+  ✧ "List" where append is list concatenation(assuming not [a])
```

Comment:
In this task, it would be enough to use `[a]`. But practising with the custom `Cons` list is a good exercise 🙂 You will more with such lists in Chapter 4.